custom_conf.py

```diff
@@ -99,7 +122,10 @@
 
     # Controls the existence of Previous / Next buttons at the bottom of pages
     # Valid options: none, prev, next, both
-    'sequential_nav': "none"
+    'sequential_nav': "none", 
+
+    #contributors information
+    'contributors': contributors
```

Comment:
This should also go into `conf.py` (as `html_context['contributors']=contributors`).